src/Symfony/Bridge/PhpUnit/Tests/SymfonyExtension.php

```diff
@@ -0,0 +1,139 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bridge\PhpUnit\Tests;
+
+use PHPUnit\Framework\Attributes\DataProvider;
+use PHPUnit\Framework\Attributes\Group;
+use PHPUnit\Framework\TestCase;
+use Symfony\Bridge\PhpUnit\Tests\Fixtures\symfonyextension\src\ClassExtendingFinalClass;
+
+class SymfonyExtension extends TestCase
+{
+    public function testExtensionOfFinalClass()
+    {
+        $this->expectNotToPerformAssertions();
```

Comment:
shouldn't this assert a deprecation instead ?

Reply:
indeed, fixed